src/feed/clients/personalize_client.py

```diff
@@ -0,0 +1,106 @@
+"""
+Client for interacting with AWS Personalize service.
+"""
+
+import logging
+from datetime import datetime, timedelta
+from typing import Dict, List, Optional
+
+from utils.aws import create_client
+
+logger = logging.getLogger(__name__)
+
+
+class PersonalizeClient:
+    """
+    Client for interacting with AWS Personalize to get personalized recommendations.
+    """
+
+    # AWS Personalize Configuration
```

Comment:
This may need to move elsewhere but for now, since this is in testing mode I may add a flag to examine production vs staging to determine right ARN.

For context, I'm not certain that we will be using filters in the future so want to avoid making env variables yet